kevin/chantal.py

```diff
@@ -193,6 +194,7 @@ def run(self, job):
             ("python3", "-u", "-m", "chantal",
              "--clone", job.build.clone_url,
              "--checkout", job.build.commit_hash,
+             "--depth", self.git_config["shallow"],
```

Comment:
the option preset in chantal is called `--clone_depth`

Reply:
Oh, true